types/array/typedArray.ts

```diff
@@ -38,19 +38,20 @@ export class TypedArrayType<T extends TypedArray>
     this.byteLength = byteLength;
   }
 
-  read(dataView: DataView, byteOffset = 0): T {
-    return this.view(dataView, byteOffset).slice() as T;
+  read(dataView: DataView, options: TypeOptions = {}): T {
+    return this.view(dataView, options).slice() as T;
```

Comment:
Copying TypedArrays like
```ts
const copy = new ArrayBuffer(source.byteLength);
new Uint8Array(copy).set(new Uint8Array(source));
```
seems to be the fastest way when compared to `structuredClone` and `slice` like we currently do for both small and large array buffers.

<img width="785" alt="bild" src="https://github.com/denosaurs/byte_type/assets/19556658/2066cbb3-cd4e-4cd1-b919-0f69160acc59">